pages/town-hall/f22.js

```diff
@@ -11,8 +11,7 @@ import Layout from '../../components/Layout';
 import TestimonialsCourseChanges from '../../public/images/town-hall/testimonials-course-changes.png';
 import TownHallFall2021Attending from '../../public/images/town-hall/town-hall-attending-f21.jpeg';
 import TownHallBanner from '../../public/images/town-hall/town-hall-banner-f22.png';
-import SpringTownHallBanner from '../../public/images/town-hall/town-hall-banner-s22.png';
-
+import SpringTownhallImage from '../../public/images/spring-2023-townhall-photo.JPG';
```

Comment:
Oops another small change, this should say spring 2022 instead of spring 2023 for the image